clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/model/discovery/EurekaInstance.groovy

```diff
@@ -97,7 +99,8 @@ class DiscoveryInstance implements Health {
       vipAddress,
       secureVipAddress,
       lastUpdatedTimestamp,
-      asgName)
+      asgName,
+      metadata?.titusTaskId)
```

Comment:
wonder if we should just have `EurekaInstance` have the whole Metadata and DataCenterInfo maps so that we don't have to encode provider specific values into this object